lib/routes/Character/components/CharacterDialog/components/blocks/BlockDicePool.tsx

```diff
@@ -143,18 +171,40 @@ export function BlockDicePoolActions(
   props: IBlockActionComponentProps<IDicePoolBlock>
 ) {
   const theme = useTheme();
+  const { t } = useTranslate();
 
+  const commands = props.block.meta.commands || [];
   return (
     <>
       <Grid item>
         <DiceMenuForCharacterSheet
-          commandGroupIds={props.block.meta.commands || []}
+          commandGroupIds={commands}
           onChange={(newCommandIds) => {
             props.onMetaChange({
               ...props.block.meta,
               commands: newCommandIds,
             });
           }}
+          render={(diceMenuProps) => (
+            <Tooltip title={commands.join(" + ")}>
+              <Link
+                component="button"
```

Comment:
Using the DiceMenuForcharacter sheet render props component again but using a `Link` instead of a `DicePool`

It's nice because with this pattern I can also decide what the button looks like without creating another component